tools/logger/convert.c

```diff
@@ -1032,55 +1032,58 @@ static int dump_ldc_info(void)
 
 int convert(void)
 {
-	struct snd_sof_logs_header snd;
+	struct snd_sof_logs_header * const logs_hdr = malloc(sizeof(*logs_hdr));
```

Comment:
It would be nice to have a symmetric free()-call if something is malloced, or use a global if one does not bother to do that.